inc/genhdr/qstrdefs.generated.h

```diff
@@ -382,6 +382,19 @@ QDEF(MP_QSTR_PITCHFORK, (const byte*)"\x33\x09" "PITCHFORK")
 QDEF(MP_QSTR_XMAS, (const byte*)"\xe2\x04" "XMAS")
 QDEF(MP_QSTR_PACMAN, (const byte*)"\x55\x06" "PACMAN")
```

Comment:
Maybe I'm being a big scaredycat, but I'd avoid names trademarked by Namco.  How about WEDGE?

p.s. I don't see microbit_const_image_pacman_obj in the diff below, am I missing something?